sklearn/linear_model/_ridge.py

```diff
@@ -1359,6 +1359,12 @@ def __sklearn_tags__(self):
         tags.classifier_tags.multi_label = True
         return tags
 
+    def _get_scorer_instance(self):
+        """Return a scorer which corresponds to what's defined in ClassiferMixin
+        parent class. This is used for routing `sample_weight`.
+        """
+        return get_scorer("accuracy")
+
```

Comment:
Some of the changes in this file seem like they were already merged in the other PR?